lib/automation.py

```diff
@@ -0,0 +1,260 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import ConfigParser
```

Comment:
I should have made this a separate commit. It's just a plain 1:1 copy of the FirefoxAutomation class.